src/main/psp.js

```diff
@@ -36,12 +37,15 @@ fluid.defaults("gpii.app.pspInApp", {
     model: {
         keyedInUserToken: "{app}.model.keyedInUserToken"
     },
+
     modelListeners: {
-        isShown: {
-            funcName: "gpii.app.hideRestartDialogIfNeeded",
-            args: ["{dialogManager}", "{change}.value"]
+        "{qssWrapper}.qss.model.isShown": {
```

Comment:
This (and all related) will probably not be needed anymore as the PSP can't be shown without the QSS being opened.